packages/design-system/src/components/EmptyState/variants/EmptyStateLarge.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { forwardRef, Ref } from 'react';
+import EmptyStatePrimitive, { EmptyStatePrimitiveProps } from '../primitive/EmptyStatePrimitive';
+
+import styles from './EmptyStateLarge.module.scss';
+
+type EmptyStateLargeProps = Omit<EmptyStatePrimitiveProps, 'illustration'> & {
+	description: string;
+};
+
+const EmptyStateLarge = forwardRef((props: EmptyStateLargeProps, ref: Ref<HTMLElement>) => {
+	return (
+		<EmptyStatePrimitive
+			{...props}
+			illustration={
+				<svg
```

Comment:
Make it as React component, it does not have to change

Reply:
Then what's the benefit of turning it into a react component and importing it here?